src/layers/ContextMenu.module.css

```diff
@@ -0,0 +1,37 @@
+/* todo: this is just copy-pasta from https://openlayers.org/en/latest/examples/popup.html, I have no idea what all this means,
+   and the popup looks ugly still, at the very minimum the background color needs to be set though
+   the popup design needs an overhaul anyway... https://github.com/graphhopper/graphhopper-maps/issues/136
+   */
+.popup {
```

Comment:
Compared to the react-map-gl wrapper OpenLayers does not provide a popup component. Therefore I used a simple div and copy-pasted some css from the OpenLayers examples for the context menu and path details popup. I'm pretty sure this can be improved...